docs/03_tutorials/tic-tac-toe-game-contract.md

```diff
@@ -293,11 +282,11 @@ void tic_tac_toe::create(const name &challenger, name &host) {
         g.challenger = challenger;
         g.host = host;
         g.turn = host;
-   });
+    });
 }
 ```
 
-### "move" Action Handler 
+### Action Handler - 'move'
```

Comment:
the other titles do not use single quotes for the action name, this one does. Is this intentional?

Reply:
no it is not intentional